themes/socialbase/assets/css/form-elements.css

```diff
@@ -2,7 +2,6 @@ label {
   display: inline-block;
   max-width: 100%;
   margin-bottom: 5px;
-  font-weight: bold;
```

Comment:
@Crazylord I'm curious why the font-weight here was removed.

Reply:
@left23 
Looks like somebody added this only in css file and after generating styles - this was removed.
I don't know if this is need or not. If yes - need add to scss file this rule.

Reply:
Oh yes, I see. I don't know if it's needed either. Thanks for the response